BTCPayServer.Tests/docker-entrypoint.sh

```
@@ -6,4 +6,4 @@ if [ ! -z "$TEST_FILTERS" ]; then
 FILTERS="--filter $TEST_FILTERS"
 fi
 
-dotnet test -c ${CONFIGURATION_NAME} $FILTERS --no-build -v n --output Normal < /dev/null
+dotnet test -c ${CONFIGURATION_NAME} $FILTERS --no-build -v n --output Normal --report-gh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)